biom/cli/table_summarizer.py

```diff
@@ -23,7 +23,7 @@ def summarize_table(table, qualitative=False, observations=False):
         compute_counts_per_sample_stats(table, qualitative)
     num_observations = len(table.ids(axis='observation'))
 
-    counts_per_sample_values = counts_per_samp.values()
+    counts_per_sample_values = list(counts_per_samp.values())
```

Comment:
If this is an `ndarray` `.tolist()` is much faster than using the `list` constructor.